altair_pandas/_core.py

```diff
@@ -205,6 +205,14 @@ def _xy(self, mark, x=None, y=None, stacked=False, **kwargs):
             .interactive()
         )
 
+        if subplots:
+            nrows, ncols = _get_layout(len(y_values), kwargs.get("layout"))
```

Comment:
I think the default layout used for matplotlib is (-1, 1), correct? We should make certain the defaults match.

Reply:
Yes, added `(-1, 1)` default.